### PR TITLE
122 customize wallet

### PR DESCRIPTION
**Pull request by OlhaD**
## Description
Add an API to update wallet

**Issue(s) addressed**
https://github.com/Greenstand/treetracker-wallet-admin-client/issues/122

**What kind of change(s) does this PR introduce?**

- [x ] Enhancement
- [ ] Bug fix
- [ ] Refactor

### CODE REVIEW

**Comment by OlhaD** [comment]
@Kpoke please review

File: server/handlers/walletHandler/schemas.js [hunk 1]
```diff
@@ -49,6 +49,14 @@ const walletPostSchema = Joi.object({
   about: Joi.string(),
 });
 
+const walletPatchSchema = Joi.object({
+  id: Joi.string().uuid().required(),
```
Comment (Kpoke) [inline server/handlers/walletHandler/schemas.js:53]: id should not be here. The id should be gotten from the url /wallet/:id

File: server/handlers/walletHandler/schemas.js [hunk 2]
```diff
@@ -49,6 +49,14 @@ const walletPostSchema = Joi.object({
...
+  name: Joi.string().required(),
```
Comment (Kpoke) [inline server/handlers/walletHandler/schemas.js:54]: Also, the name should not be updateable at this moment

**Review by Kpoke — CHANGES_REQUESTED** [review]

File: server/handlers/walletHandler/schemas.js (hunk 1 above)
Comment (OlhaD) [reply server/handlers/walletHandler/schemas.js]: I removed it. Maybe I should better take it from the request (from the token) like in walletPost? It doesn't have id in the URL and I would expect Patch request to look similar to Post

**Pull request merged**